subprojects/packagefiles/pcre/meson_options.txt

```diff
@@ -1 +1,7 @@
-option('shared_lib', type : 'boolean', value : false)
```

Comment:
Not strictly part of the stated purpose of this PR.

On the other hand, thank you! :D :D :D

Reply:
Yeah that's probably just a cmake option to deal with cmake's inability to have both library types. At least not easily.